pod_test.go

```diff
@@ -280,8 +280,8 @@ func TestPodCollector(t *testing.T) {
 				},
 			},
 			want: metadata + `
-				kube_pod_info{host_ip="1.1.1.1",namespace="ns1",pod="pod1",node="node1",pod_ip="1.2.3.4"} 1
-				kube_pod_info{host_ip="1.1.1.1",namespace="ns2",pod="pod2",node="node2",pod_ip="2.3.4.5"} 1
+				kube_pod_info{controllers="<none>",host_ip="1.1.1.1",namespace="ns1",pod="pod1",node="node1",pod_ip="1.2.3.4"} 1
```

Comment:
@brancz  Though CreatedByAnnotation represents the key used to store the spec(json)
used to create the resource ,controllers named come from` kubeclt describe po xxx`,maybe we need to think about the consistency with other projects